downstream/modules/platform/proc-operator-create_crs.adoc

```diff
@@ -19,3 +48,9 @@ This enables upgrading a specific deployment, like a controller, without updatin
 
 The recommended approach however, is to upgrade the operator and use the default image values. 
 ====
+
+.Verification
+Go to your {OperatorPlatformNameShort} deployment and click btn:[All instances] to verify if all instances deployed correctly.
```

Comment:
You might consider using "Navigate to" rather than "Go to" here. Up to you, though!

Reply:
"to verify whether all instances have deployed correctly." This might be slightly clearer.